openedx/core/djangoapps/content/learning_sequences/api/tests/test_outlines.py

```diff
@@ -1716,6 +1718,216 @@ def test_processor_only(
             removed_usage_keys = processor.usage_keys_to_remove(full_outline)
             assert len(removed_usage_keys) == expected_values_dict[learner_to_verify.username]
 
+@ddt.ddt
+class CohortPartitionGroupsTestCase(OutlineProcessorTestCase):  # lint-amnesty, pylint: disable=missing-class-docstring
```

Comment:
Do we need these linter directives here?

Reply:
We do not! Removed.